tmail-backend/jmap/extensions/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
What is this file?

Reply:
configuration for mockito to able to mock final `FirebaseMessagingException` class.